src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java

```diff
@@ -696,14 +819,803 @@ public boolean validatePortRange(Integer portNumber) {
+				break;
+			case STILL_IMAGE_DELAY:
+				Integer stillImageDelay = DecoderConstant.DEFAULT_STILL_IMAGE_DELAY;
+				try {
+					stillImageDelay = Integer.parseInt(value);
+					if (stillImageDelay < DecoderConstant.MIN_STILL_IMAGE_DELAY) {
+						stillImageDelay = DecoderConstant.MIN_STILL_IMAGE_DELAY;
+					}
+					if (stillImageDelay > DecoderConstant.MAX_STILL_IMAGE_DELAY) {
+						stillImageDelay = DecoderConstant.MAX_STILL_IMAGE_DELAY;
+					}
+				} catch (Exception e) {
+					if (logger.isWarnEnabled()) {
+						logger.warn("Invalid still image delay value");
+					}
+				}
+				cachedDecoderConfig.setStillImageDelay(value);
+				this.cachedDecoderConfigs.set(decoderID, cachedDecoderConfig);
+				addAdvanceControlProperties(advancedControllableProperties,
+						createNumeric(stats, decoderControllingGroup + DecoderControllingMetric.STILL_IMAGE_DELAY.getName(), stillImageDelay.toString()));
+				populateApplyChangeAndCancelButtonForDecoder(stats, advancedControllableProperties, decoderID);
+				populateLocalExtendedStats(stats, advancedControllableProperties);
+				break;
+			case SYNC_MODE:
+				boolean isEnableSyncMode = mapSwitchControlValue(value);
+				String enableSyncMode = SyncMode.ENABLE_SYNC_MODE.getName();
+				if (!isEnableSyncMode) {
+					enableSyncMode = SyncMode.DISABLE_SYNC_MODE.getName();
+				}
+				cachedDecoderConfig.setEnableBuffering(enableSyncMode);
+				this.cachedDecoderConfigs.set(decoderID, cachedDecoderConfig);
+				addAdvanceControlProperties(advancedControllableProperties, createSwitch(stats, decoderControllingGroup + DecoderControllingMetric.SYNC_MODE.getName(), Integer.parseInt(value),
+						DecoderConstant.DISABLE, DecoderConstant.ENABLE));
+				populateApplyChangeAndCancelButtonForDecoder(stats, advancedControllableProperties, decoderID);
+				populateLocalExtendedStats(stats, advancedControllableProperties);
+				break;
+			case BUFFERING_MODE:
+				BufferingMode bufferingMode = BufferingMode.getByUiName(value);
+				cachedDecoderConfig.setBufferingMode(bufferingMode.getApiName());
+				this.cachedDecoderConfigs.set(decoderID, cachedDecoderConfig);
+				populateDecoderControlBufferingMode(stats, advancedControllableProperties, cachedDecoderConfig, decoderID);
+				populateApplyChangeAndCancelButtonForDecoder(stats, advancedControllableProperties, decoderID);
+				populateLocalExtendedStats(stats, advancedControllableProperties);
+				break;
+			case BUFFERING_DELAY:
+				Integer bufferingDelay = DecoderConstant.MIN_BUFFERING_DELAY;
+				try {
+					bufferingDelay = Integer.parseInt(value);
+					if (bufferingDelay < DecoderConstant.MIN_BUFFERING_DELAY) {
+						bufferingDelay = DecoderConstant.MIN_BUFFERING_DELAY;
+					}
+					if (bufferingDelay > DecoderConstant.MAX_BUFFERING_DELAY) {
+						bufferingDelay = DecoderConstant.MAX_BUFFERING_DELAY;
+					}
+				} catch (Exception e) {
+					if (logger.isWarnEnabled()) {
+						logger.warn("Invalid buffering delay value");
```

Comment:
Also log for catched exception.
logger.warn("Invalid buffering delay value",e);

Reply:
i fixed it